library/src/main/java/com/otaliastudios/zoom/ZoomApi.kt

```diff
@@ -200,13 +222,16 @@ interface ZoomApi {
     fun setTransformation(@Transformation transformation: Int, gravity: Int)
 
     /**
-     * Sets the policy to use when content is smaller than the container.
-     * Defaults to [SMALLER_POLICY_CENTER]
-     * which means that the content will be centered.
+     * Sets the content alignment. Can be any of the constants defined in [com.otaliastudios.zoom.Alignment].
```

Comment:
Is it really necessary to mention the `com.otaliastudios.zoom.Alignment` class? The `@Annotation` annotation on the parameter should already specify this and if I'm not mistaking the IDE will correct you (the dev using this library) too.